src/utils/validation/customerInstitutionValidation.ts

```diff
@@ -39,12 +43,13 @@ export const customerInstitutionValidationSchema = Yup.object<YupShape<CustomerI
   doiAgent: Yup.object().when('canAssignDoi', {
     is: true,
     then: () =>
-      Yup.object<YupShape<DoiAgent>>({
+      Yup.object<YupShape<ProtectedDoiAgent>>({
         username: Yup.string().required(customerErrorMessage.doiNameRequired),
         prefix: Yup.string()
           .matches(/^10.(\d){4,9}$/, customerErrorMessage.doiPrefixInvalid)
           .required(customerErrorMessage.doiPrefixRequired),
         url: Yup.string().required(customerErrorMessage.doiUrlRequired),
+        password: Yup.string().optional(), // Password in validated inline, on the Field component
```

Comment:
Fikk ikke til å validere passord her på en god måte fordi Yup insisterer på at tom streng blir `undefined`, som jo skal være lov. Finner ikke ut hvorfor det er et problem her, da der virker som det skal andre plasser, men bruker ikke mer tid på det nå da dette uansett bare er for interne brukere.

Plana var noe slik:
```
.test(
          'doi-password-test',
          i18n.t('translation:feedback.validation.is_required', {
            field: i18n.t('translation:basic_data.institutions.doi_password'),
          }),
          (value) => value === undefined || (value && value.length > 0); 
        )
```